redash/query_runner/pg.py

```diff
@@ -70,6 +70,11 @@ def configuration_schema(cls):
                 "dbname": {
                     "type": "string",
                     "title": "Database Name"
+                },
+                "sslmode": {
+                   "type": "string",
+                   "title": "SSL Mode",
+                   "default": "require"
```

Comment:
Should the default mode be `require` without a semi-loud doc update? I know this would be fairly confusing if I didn't have ssl server side. What about `prefer`?